### DISCUSSION TITLE
Resume cannot exceed 320 characters missing character count

### DISCUSSION THREAD

**Discussion by dinkydani** [paircodedev/sandbox#12] [Bug Report] [answered]
This field should show a character count otherwise it's very difficult to know how long the text is :) 

<img width="796" alt="image" src="https://user-images.githubusercontent.com/3687223/120090562-3b2afd80-c136-11eb-872e-dffe4fdea634.png">

**Comment by Necmttn** [comment, marked as answer]
Woop fixed this part as well! 
![image](https://user-images.githubusercontent.com/5212808/120273771-85041700-c2e1-11eb-9e83-5f3bf7159d99.png)
now the top limit is 5000 char!

**Reply by Necmttn** [reply]
@dinkydani Thank you for all the bug reports 💛 🧑‍💻 😻
